### ISSUE TITLE
Please add tensorflow, keras, pytorch, etc. to the requirements.txt file

### ISSUE THREAD

**Issue by sbbogdan** [udacity/RL-Quadcopter#5]
For the deep learning nanodegree we are going to have to use at least one, or multiple of these libraries. It makes no sense for their presence not to be in the requirements.txt file, or even mentioned anywhere in the instructions.

It would be nice if you could:
a) Add an README.md step beneath 'Python Packages' that explicitly requires students to open the requirements.txt file
b) Have commented out lines for popular packages like tensorflow, keras, pytorch, etc. and ask students to review what they think they will use, and then uncomment such out prior to proceeding. Realistically, hard drive space is so cheap and plentiful these days that these libraries should not be a significant space burden and it is probably just easier to include at least keras, pandas, and tensorflow by default. Students can uninstall what they don't need.

**Comment by ablacklama** [comment]
pandas/h5py too.

But tensorflow can't just be installed with pip. You need Cuda/cudnn too. So you couldn't just automatically install it

**Comment by sbbogdan** [comment]
+1 to pandas and h5py. Those were needed for this project.

Tensorflow can be installed with pip, that's what I did. It just won't be GPU accelerated AFAIK.

**Comment by ablacklama** [comment]
you're completely right. I honestly forget that cpu tensorflow even exists sometimes... I removed it from my comment
